tests/test_enterprise/test_tasks.py

```diff
@@ -180,3 +194,117 @@ def test_send_enterprise_email_notification(self, mock_send_notification, mock_e
+            recipients=mock_recipients,
+            trigger_properties={
+                'contact_admin_link': mock_admin_mailto,
+                'enterprise_customer_name': self.enterprise_customer.name,
+                'catalog_content_count': mock_catalog_content_count,
+                'act_by_date': act_by_date.strftime('%B %d, %Y'),
+            },
+        )]
+        mock_braze_api_client().send_campaign_message.assert_has_calls(calls)
+
+    @mock.patch('enterprise.tasks.EnterpriseCatalogApiClient', return_value=mock.MagicMock())
+    @mock.patch('enterprise.tasks.BrazeAPIClient', return_value=mock.MagicMock())
+    def test_send_group_membership_removal_notification(self, mock_braze_api_client, mock_enterprise_catalog_client):
+        """
+        Verify send_group_membership_removal_notification hits braze client with expected args
+        """
+        EnterpriseGroupMembershipFactory(
+            group=self.enterprise_group,
+            pending_enterprise_customer_user=PendingEnterpriseCustomerUserFactory(),
+            enterprise_customer_user=None,
+        )
+        EnterpriseGroupMembershipFactory(
+            group=self.enterprise_group,
+            pending_enterprise_customer_user=None,
+            enterprise_customer_user__enterprise_customer=self.enterprise_customer,
+            activated_at=datetime.now()
+        )
+        admin_email = 'edx@example.org'
+        mock_recipients = [self.pending_enterprise_customer_user.user_email,
+                           mock_braze_api_client().create_recipient.return_value]
+        mock_admin_mailto = f'mailto:{admin_email}'
+        mock_braze_api_client().generate_mailto_link.return_value = mock_admin_mailto
+        mock_braze_api_client().create_recipient_no_external_id.return_value = (
+            self.pending_enterprise_customer_user.user_email)
+        mock_catalog_content_count = 5
+        mock_enterprise_catalog_client().get_catalog_content_count.return_value = (
+            mock_catalog_content_count)
+        membership_uuids = EnterpriseGroupMembership.objects.values_list('uuid', flat=True)
+        catalog_uuid = uuid.uuid4()
+        send_group_membership_removal_notification(
+            self.enterprise_customer.uuid,
+            membership_uuids,
+            catalog_uuid,
+        )
+        calls = [mock.call(
+            BRAZE_GROUPS_REMOVAL_EMAIL_CAMPAIGN_ID,
+            recipients=mock_recipients,
+            trigger_properties={
+                'contact_admin_link': mock_admin_mailto,
+                'enterprise_customer_name': self.enterprise_customer.name,
+                'catalog_content_count': mock_catalog_content_count,
+            },
+        )]
+        mock_braze_api_client().send_campaign_message.assert_has_calls(calls)
+
+    @mock.patch('enterprise.tasks.BrazeAPIClient', return_value=mock.MagicMock())
+    def test_sso_configuration_oauth_orchestration_email(self, mock_braze_client):
```

Comment:
Should we have a test for if something goes wrong?

Reply:
I moved this test over to the more appropriate testing file but it is unrelated to this work. the sso changes should be in a separate pr for the fix we discussed.